azure-pipelines.yml

```diff
@@ -77,13 +77,13 @@ jobs:
 
       # Test the binary
       # For now only on Ubn16.04
-      - script: |
-          ls -al
-          chmod 777 ./blobfusetests
-          $(Build.BinariesDirectory)/blobfusetests --gtest_filter=-BlobClientWrapperTest.*
-        workingDirectory: $(Build.BinariesDirectory)
-        condition: eq(variables.container_image, 'ubuntu-18.04')
-        displayName: "GTest on Ubn-18"
+      #- script: |
```

Comment:
What are these tests? Should we make an item to re-enable them?

Reply:
These are kind of UT written using google test suite for the sdk. However they are sparse in nature (not very inclusive) so its ok to ignore them.